ios_widget_catalogue_compare/ios/Runner/OverlayFlutterViewController.swift

```diff
@@ -0,0 +1,150 @@
+import SwiftUI
+
+
+/// The initial `UIViewController` defined in `Main.storyboard` uses this class.
+/// It is overlayed on top of the Flutter controls that are defined in `../lib`
+/// It will relay touch events to Flutter, so that the same touch event will reproduce
+/// with both Flutter and Swift.
+
+@available(iOS 14.0, *)
+class OverlayFlutterViewController: FlutterViewController, FlutterStreamHandler, ObservableObject {
+  
+  // Widget catalogue
+  @Published var controlKey: String = ""
+    
+  // SUI Overlay
+  lazy var overlaySUIView: OverlaySwiftUIView = setOverlaySUIView()
+  func setOverlaySUIView() -> OverlaySwiftUIView {
+    OverlaySwiftUIView(controller: self)
+  }
+  var originalOverlayCenter: CGPoint = .zero
+  
+  lazy var suiController: UIViewController = setSUIController()
+  func setSUIController() -> UIViewController {
+    UIHostingController(rootView: OverlaySwiftUIView(controller: self))
+  }
+
+  // UI Controls
+  var dropDownButton: UIButton = UIButton(frame: CGRect(x: 13, y: 50, width: 300, height: 40))
+
+  var xSlider: UISlider = UISlider()
+  var ySlider: UISlider = UISlider()
+  var alphaSlider: UISlider = UISlider()
+  
+  lazy var slidersStackView: UIStackView = setSlidersStackView()
+  func setSlidersStackView() -> UIStackView {
+    UIStackView(arrangedSubviews: [xSlider, ySlider, alphaSlider])
+  }
+  
+  // Flutter Stream Handler
+  var eventSink: FlutterEventSink?
+  
+  lazy var eventChannel: FlutterEventChannel = setEventChannel()
+  func setEventChannel() -> FlutterEventChannel {
+    FlutterEventChannel(name: "overlay_ios.flutter.io/responder", binaryMessenger: self as! FlutterBinaryMessenger)
+  }
+  
+  //MARK: App cycle methods
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    
+    xSlider.minimumValue = -200
+    xSlider.maximumValue = 200
+    xSlider.isContinuous = true
+    xSlider.addTarget(self, action: #selector(xSliderChanged), for: .valueChanged)
+    
+    ySlider.minimumValue = -200
+    ySlider.maximumValue = 200
+    ySlider.isContinuous = true
+    ySlider.addTarget(self, action: #selector(ySliderChanged), for: .valueChanged)
+    
+    alphaSlider.minimumValue = 0
+    alphaSlider.maximumValue = 1
+    alphaSlider.value = 1
+    alphaSlider.isContinuous = true
+    alphaSlider.addTarget(self, action: #selector(alphaSliderChanged), for: .valueChanged)
+    
+    slidersStackView.axis = .vertical
+    slidersStackView.distribution = .equalSpacing
+    slidersStackView.center = self.view.center
+    slidersStackView.backgroundColor = UIColor.lightGray.withAlphaComponent(0.6)
+    slidersStackView.isLayoutMarginsRelativeArrangement = true
+    slidersStackView.directionalLayoutMargins = NSDirectionalEdgeInsets(top: 10, leading: 20, bottom: 10, trailing: 20)
+    slidersStackView.spacing = UIStackView.spacingUseSystem
```

Comment:
can we block touches here too? Otherwise, it might be confusing if a drag misses the sliders but end up moving the picker behind it